pkg/store/mastership/store.go

```diff
@@ -126,6 +131,15 @@ func (s *atomixStore) NodeID() cluster.NodeID {
 	return s.nodeID
 }
 
+func (s *atomixStore) GetTerm(deviceID topodevice.ID) (Term, error) {
+	if election, found := s.elections[deviceID]; found {
+		term := election.getTerm()
+		return term, nil
+	}
+
+	return 0, fmt.Errorf("device mastership does not exist for device %s", deviceID)
```

Comment:
@kuujo 
Can we assume the term starts from one and zero represents nothing or its initial value ?

Reply:
The term for a real master is always `> 0`. The term will only be `0` if no master has ever been elected.

Reply:
So it makes sense then.